examples/ensure-name-substring/advanced/Kptfile

```diff
@@ -0,0 +1,8 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/ensure-name-substring:unstable
```

Comment:
will unstable be removed in the final docs?

Reply:
`unstable` tag will be switched to the corresponding version tag in their release branches.
We will keep using unstable in the master branch.